sidekick/sidekick-broker/sidekick-broker-app/src/main/java/com/runsidekick/broker/SidekickBrokerApplication.java

```diff
@@ -15,7 +19,13 @@
 )
 @RestController
 @EnableScheduling
-public class SidekickBrokerApplication {
+public class SidekickBrokerApplication implements CommandLineRunner {
+
+    @Value("${phone-home.enabled:true}")
```

Comment:
Not %100 sure but if we use our property source infra from Thundra infra, `-` character might be problematic when we want to pass same config as environment variable. Because, `-` is not a valid character for env var.